pkg/runner/runner.go

```diff
@@ -158,20 +154,31 @@ func RunBundle(action string, ns string, bundleName string, sandboxRole string,
 	}
 	_, err = k8scli.Client.CoreV1().Pods(ns).Create(pod)
 	if err != nil {
-		log.Errorf("Failed to create pod: %v", err)
-		// TODO: return ErrorPodCreationFailed
-		return
+		return "", err
 	}
-	fmt.Printf("Successfully created pod [%v] to %s [%v] in namespace [%v]\n", pn, ec.Action, bundleName, ns)
+	fmt.Printf("Successfully created pod [%v] to %s [%v] in namespace [%v]\n", podName, ec.Action, bundleName, ns)
 
 	if printLogs {
-		printBundleLogs(pn, ns, action)
+		printBundleLogs(podName, ns, action)
 	}
 
-	// TODO: return nil
 	return
 }
 
+func GetPodStatus(namespace string, podName string) string {
+	k8scli, err := clients.Kubernetes()
+	if err != nil {
+		panic(err.Error())
+	}
+	pod, err := k8scli.Client.CoreV1().Pods(namespace).Get(podName, metav1.GetOptions{})
+	if err != nil {
+		log.Errorf("Error getting pod [%v] status: %v", podName, err)
+		return ""
```

Comment:
feels like this should return something other than empty string.

Reply:
Going to include an error return as well